ambari-infra-manager/src/test/java/org/apache/ambari/infra/json/FsPermissionToStringConverterTest.java

```diff
@@ -0,0 +1,38 @@
+package org.apache.ambari.infra.json;
+
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+import org.apache.hadoop.fs.permission.FsPermission;
+import org.junit.Test;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Seems misplaced, usually the license is at the top of files, although this does not cause RAT failure.